cmd/exec.go

```diff
@@ -42,27 +42,26 @@ func execRun(cmd *cobra.Command, args []string) error {
 		return errors.New("must specify command to run")
 	}
 
-	service := strings.ToLower(args[0])
-	if err := validateService(service); err != nil {
-		return err
-	}
-
 	command := commandPart[0]
 
 	var commandArgs []string
 	if len(commandPart) > 1 {
 		commandArgs = commandPart[1:]
 	}
 
-	secretStore := store.NewSSMStore()
-	secrets, err := secretStore.List(service, true)
-	if err != nil {
-		return err
-	}
-
 	env := environ(os.Environ())
-	for _, secret := range secrets {
-		env.Set(strings.ToUpper(key(secret.Meta.Key)), *secret.Value)
+	secretStore := store.NewSSMStore()
+	for _, service := range args {
+		if err := validateService(service); err != nil {
+			return err
+		}
+		secrets, err := secretStore.List(service, true)
+		if err != nil {
+			return err
+		}
+		for _, secret := range secrets {
+			env.Set(strings.ToUpper(key(secret.Meta.Key)), *secret.Value)
+		}
```

Comment:
I'm not sure.  I'd expect a failure here.

Worst case scenario is you have two variables, one which overrides the other when set but the override doesn't get set properly.  You continue happily with the original, unaware that anything's gone wrong.  If you don't fail everything that it will probably get discovered in production and via logs instead of failing to start the first time.